static/src/js/selectors/earthdataEnvironment.js

```diff
@@ -1,10 +1,27 @@
+import { createSelector } from 'reselect'
+import { parse } from 'qs'
+
 import { getApplicationConfig } from '../../../../sharedUtils/config'
 
 /**
- * Retrieve current CMR environment from Redux
+ * Retrieve the earthdataEnvironment param from the router location in Redux
  * @param {Object} state Current state of Redux
  */
-export const getEarthdataEnvironment = (state) => {
+export const getEarthdataEnvironmentFromUrl = (state) => {
+  const { router = {} } = state
+  const { location = {} } = router
+  const { search } = location
```

Comment:
do we need to default this or will parse handle an undefined/null value?

Reply:
Search is defaulted in the store to an empty string, but doing that here wouldn't hurt